funnel/forms/space.py

```diff
@@ -55,6 +55,9 @@ class ProposalSpaceForm(forms.Form):
     review_team = QuerySelectField(u"Review Team", validators=[forms.validators.DataRequired(__(u"Please select a team"))],
         query_factory=profile_teams, get_label='title', allow_blank=False,
         description=__(u"Reviewers can see contact details of proposers, but can’t change settings"))
+    checkin_team = QuerySelectField(u"Checkin Team", validators=[forms.validators.DataRequired(__(u"Please select a team"))],
+        query_factory=profile_teams, get_label='title', allow_blank=False,
+        description=__(u"Checkin team can scan badge to checkin users to event"))
```

Comment:
"Team members can check in users at an event"

Reply:
Fixed